### ISSUE TITLE
Handle `INTEGER` data type in the backend.

### ISSUE THREAD

**Issue by kgodey** [centerofci/mathesar#379]

**Comment by mathemancer** [comment]
Note that suggesting an integer type has the potential to confuse a user who doesn't understand the difference between `INTEGER` and `DECIMAL`/`NUMERIC` types.  Since there's no distinction in spreadsheets, it wouldn't surprise me if we get such users.

**Comment by kgodey** [comment]
@mathemancer Makes sense. We should think carefully about when it might make sense to suggest `INTEGER`, the answer right now might be "never".
